Added USAGE_DATA.md (#247)
redhat-developer/vscode-knative: USAGE_DATA.md

## [knative Extension](https://github.com/redhat-developer/vscode-knative)

### Usage Data

> * when extension is activated

Reviewer: ```suggestion
* When the extension is activated
```

Author: done

> * when a command contributed by extension is executed

Reviewer: ```suggestion
* When following command(s) contributed by extension is executed
```

Author: done

    * command's ID
    * command's error message (in case of exception)

>     * command's specific data like knative, function version and to check which command user has used.

Reviewer: ```suggestion
    * command's specific data like kn version, fn version and to check which command user has used.
```

Author: done

> * when extension is deactivated

Reviewer: ```suggestion
* When the extension is deactivated
```

> * Command which send data to telemetry.

Reviewer: ```suggestion
* Following are the commands which send data to telemetry.
```
